src/FormElement/ScheduleElement/AnnuallyFields.php

```diff
@@ -0,0 +1,145 @@
+        $this->initUtils();
+
+        $this->months = [
+            'JAN' => $this->translate('Jan'),
+            'FEB' => $this->translate('Feb'),
+            'MAR' => $this->translate('Mar'),
+            'APR' => $this->translate('Apr'),
+            'MAY' => $this->translate('May'),
+            'JUN' => $this->translate('Jun'),
+            'JUL' => $this->translate('Jul'),
+            'AUG' => $this->translate('Aug'),
+            'SEP' => $this->translate('Sep'),
+            'OCT' => $this->translate('Oct'),
+            'NOV' => $this->translate('Nov'),
+            'DEC' => $this->translate('Dec')
+        ];
+    }
+
+    public function onRegistered(Form $form)
+    {
+        $form->on(Form::ON_SENT, function ($form) {
+            $this->isAutoSubmitted = ! $form->hasBeenSubmitted();
+        });
+    }
+
+    /**
+     * Set the default month to be activated
+     *
+     * @param string $default
+     *
+     * @return $this
+     */
+    public function setDefault(string $default): self
+    {
+        // Attributes are registered far before the initialization of this element!
+        if (! empty($this->months) && ! isset($this->months[strtoupper($this->default)])) {
+            throw new InvalidArgumentException(sprintf('Invalid month provided: %s', $default));
+        }
+
+        $this->default = strtoupper($default);
+
+        return $this;
+    }
+
+    protected function assemble()
+    {
+        $this->getAttributes()->set('id', $this->protectId('annually-fields'));
+
+        $fieldsSelector = new FieldsRadio('month', [
+            'class'     => 'autosubmit sr-only',
+            'value'     => $this->default,
+            'options'   => $this->months,
+            'protector' => function ($value) {
+                return $this->protectId($value);
+            }
+        ]);
+        $this->registerElement($fieldsSelector);
+
+        $runsOnThe = $this->getPopulatedValue('runsOnThe', 'n');
+        $this->addElement('checkbox', 'runsOnThe', [
+            'class' => 'autosubmit',
+            'value' => $runsOnThe
+        ]);
+
+        $checkboxControls = HtmlElement::create('div', ['class' => 'toggle-slider-controls']);
+        $checkbox = $this->getElement('runsOnThe');
+        $checkbox->prependWrapper($checkboxControls);
+        $checkboxControls->addHtml($checkbox, HtmlElement::create('span', null, $this->translate('On the')));
+
+        $annuallyWrapper = HtmlElement::create('div', ['class' => 'annually']);
+        $checkboxControls->prependWrapper($annuallyWrapper);
+        $annuallyWrapper->addHtml($fieldsSelector);
+
+        $notes = HtmlElement::create('div', ['class' => 'note']);
+        $notes->addHtml(
+            HtmlString::create($this->translate('Use ')),
+            new Icon('arrow-left'),
+            HtmlString::create(' / '),
+            new Icon('arrow-right'),
+            HtmlString::create($this->translate(' keys to choose a month by keyboard.'))
+        );
```

Comment:
It's now translatable, but: Imagine how a translator sees this:

* `Use `
* ` keys to choose a month by keyboard.`

Take a look [here](https://translate.icinga.com/translate/icinga/icingadb-web/de/?q=) and imagine these intermingled with others in between. Yeah, they are grouped by location and appearance in files, but we shouldn't make it harder than required.

Construct messages like this with sprintf syntax:

`FormattedString::create($this->translate('Use %s / %s keys to choose a month by keyboard.'), <icon>, <icon>)`

Reply:
(v)